content/includes/header.html

```diff
@@ -47,6 +47,7 @@
 <meta name="msapplication-TileColor" content="#144073">
 
 <!-- CSS -->
+<link rel="stylesheet" href="/generated/style.css">
```

Comment:
Do any of our bundles _not_ require `style.css`?  Then a followup: if they all require `style.css`, does every bundle require every part of `style.css`? Otherwise `style.css` seems like a dumping ground for all styles and a first-time visitor  may end up downloading a large `style.css` file and only need to use some small percentage of it.